pkgs/development/tools/golangci-lint/deps.nix

```diff
@@ -0,0 +1,498 @@
+# file generated from Gopkg.lock using dep2nix (https://github.com/nixcloud/dep2nix)
+[
```

Comment:
Why do you need this file? Vendor dir is already in git of golangci-lint: no need to fetch dependencies